lib/portlayer/exec/config.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/vmware/vic/lib/portlayer/event"
 )
 
-var VCHConfig Configuration
+var Config Configuration
```

Comment:
iirc @caglar10ur had a change to add the "VCH" prefix -- I believe he was concerned about the proliferation of "config" vars in the codebase -- might double check